docker/gaffer-integration-tests/docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Update store properties files to point to the location of the Accumulo store to test against:
+cd ~/store-implementation/accumulo-store/src/test/resources/
```

Comment:
docker-compose up fails because the repo is cloned into /tmp/gaffer rather than /root. You should cd to /tmp/gaffer instead